packages/osmap.yaml

```diff
@@ -18,3 +18,6 @@ Centos:
   snaps:
     package:
 
+Ubuntu:
+  snaps:
+    packages: ['snapd', 'fuse',]
```

Comment:
@abele if you add `fuse` to `defaults.yam` then osmap.yaml does not need to be updated.  Every linux should have fuse (and squashfuse) packages.

Reply:
@noelmcloughlin, maybe I'm wrong, but it seems `squashfuse` is a [100% Ubuntu thing](https://launchpad.net/ubuntu/+source/squashfuse/0.1.100-0ubuntu2) at the moment, written to fix these `snaps-in-docker` issues. I got there trying to find, precisely, other distros' `squashfuse` packages, which I couldn't find. Not even for Debian

Reply:
I know `snapd` and `fuse` are widely available so would suit `defaults.yaml` otherwise you have to repeat same values for every distro in `osmap.yaml`.

I'm inclined to agree that squashfuse is not widely available so cannot belong to `defaults.yaml`.  Its available in Fedora (http://rpm.pbone.net/index.php3?stat=3&search=squashfuse) but not in EL7 or Suse by default.  

@abele maybe this is only Ubuntu issue so I'm okay to only fix in Ubuntu.

Reply:
@noelmcloughlin only `fuse` package is required.
I have removed changes to `osmap.yaml` and added `packages: ['snapd', 'fuse',]` to `defaults.yaml`.